nox.py

```diff
@@ -320,3 +320,12 @@ def mark_if_necessary(requirement):
         for requirement in sorted(requirements, key=lambda s: s.lower()):
             if not requirement.startswith('#'):
                 f.write(requirement)
+
+
+def session_readmegen(session):
+    session.install('-r', 'testing/requirements-dev.txt')
+
+    in_files = list(list_files('.', 'README.rst.in'))
```

Comment:
Oh - I meant `README.rst.yaml` to make clear that it's a yaml file (and for syntax highlighting, etc). The fact that it's an input to the resulting `rst` would be implied.